Learn-Level-1/Challenge-5/main.py

```diff
@@ -0,0 +1,50 @@
+"""
+Challenge 5: Path Payment
+"""
+from stellar_sdk import Asset, Server, Keypair, TransactionBuilder, Network
+import requests
+
+# 1. Load Keys
+server = Server("https://horizon-testnet.stellar.org")
+stellar_quest_keypair = Keypair.from_secret("SDGWX6D32AGE6HNSTA5PUCJP2MQCAHTYJOMCTSJETUXGHON2ULUEAWRX")
+quest_account_pub_key = stellar_quest_keypair.public_key
+quest_account_priv_key = stellar_quest_keypair.secret
+
+# 2. Path
+path = [
+    Asset("USDT", "GDMUX3LEQ2HBJDMHTYNSR23YC3JZWTWY63KRMZ7B4Z5II6X5BGB73UWS"),
+    Asset("BTC", "GDMUX3LEQ2HBJDMHTYNSR23YC3JZWTWY63KRMZ7B4Z5II6X5BGB73UWS"),
+]
+
+# 3. Transaction
+
+print("Building Transaction...")
+
+base_fee = server.fetch_base_fee()
+stellar_account = server.load_account(quest_account_pub_key)
+print('IF IT FAILS CHECK THE TESTNET ORDERBOOK AND CHANGE DEST ASSET')

```

Comment:
I think this print statement can be solved much better. One solution I can think of at the moment is to verify the path isn't empty and if it is to print an error message

Reply:
I wasted half an hour debugging "op_too_few_offers" error lol, so printed this to warn the user